client/src/languages/external.ts

```diff
@@ -0,0 +1,190 @@
+import {h} from 'maquette';
+import * as Langs from '../definitions/languages'; 
+import * as Graph from '../definitions/graph'; 
+import * as Values from '../definitions/values'; 
+import {printPreview} from '../editors/preview'; 
+import {createEditor} from '../editors/editor';
+import {Md5} from 'ts-md5';
+import axios from 'axios';
+
+declare var DATASTORE_URI: string;
+
+// ------------------------------------------------------------------------------------------------
+// rPython plugin
+// ------------------------------------------------------------------------------------------------
+
+/// A class that represents a Python block. All blocks need to have 
+/// `language` and Python also keeps the Python source we edit and render
```

Comment:
Can we please update the above comment to be up-to-date? It's not just R/Python anymore...